pkg/controller/logstorage/esgateway.go

```diff
@@ -44,7 +44,7 @@ func (r *ReconcileLogStorage) createEsGateway(
 		return reconcile.Result{}, false, nil
 	}
 
-	esInternalCertSecret, err := utils.GetSecret(ctx, r.client, relasticsearch.InternalCertSecret, render.ElasticsearchNamespace)
+	esInternalCertSecret, err := utils.GetSecret(ctx, r.client, relasticsearch.InternalPublicCertSecret, render.ElasticsearchNamespace)
```

Comment:
This name seems to contradict itself. Internal and Public are opposites in my mind.

Reply:
I was thinkin of it as `internal` in the sense that it is used internally for components to communicate directly with elasticsearch and it should not be used/overrided by an user, and the `PublicCert` parts indicating this secret contains only the public certificate portion of a TLS certificate secret.

Maybe ESPublicCertSecret would make more sense, to indicate this is directly related to elasticsearch and not WSGateway, what do you think?
Another suggestion would be to add to the secrets that contains the key a `PrivateCert` variable name part.

Reply:
I think I'd rather call ones that contain the key in addition to the cert to be `KeyCert` or `CertKey`.
I think Key is less ambiguous compared to Private, WDYT?

Reply:
I'm going to retract my previous comment. I think it doesn't help because both the public and private parts are both called keys.
I do like your suggestion to add PrivateCert to the secrets that contain the private key.

I wonder if we should drop the "Internal" part of the name with this change and add "External" to secrets that might be externally provided. We should just assume secrets are Internal unless otherwise specified.
Maybe we shouldn't be trying to encode the Internal/External in the name though because after all these are just names, it is how we read and use them that determines if they are Internal/External.

Reply:
I agree on PrivateCert, that would make clear on what this variables are pointing to.

For `External`, it sounds to me that it **must** be provided instead of **could**, so I don't think we should add it (if we must identify those, something like `Overridable/Customizable` might be better, but it would make a necessary step to rename these every time there is a change if a secret may be customized or not, which does not seem ideal to me). 

Dropping `Internal` and letting the default be that secrets are internal makes sense to me.

With these changes, we would have these names as:
```
relasticsearch.InternalCertSecret -> relasticsearch.PrivateCertSecret
relasticsearch.InternalPublicCertSecret -> relasticsearch.PublicCertSecret
```

That makes sense to me in identifying the first one as the elasticsearch private secret and the second one is the public version of the same secret.

Any thoughts?

Reply:
Restored to original names. Related naming modifications will be moved to another change.